programs/ssl/ssl_client2.c

```diff
@@ -1890,7 +2044,7 @@ int main( int argc, char *argv[] )
     }
 #endif /* MBEDTLS_SSL_DTLS_CONNECTION_ID */
 
-#if defined(MBEDTLS_ECP_C)
+#if defined(MBEDTLS_ECP_C) && !defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
```

Comment:
Why was this guard added? I understand that the code that's now guarded is TLS 1.2 specific, but we want the code to build with both TLS 1.2 and TLS 1.3 enabled.